app/src/main/java/com/github/se/wanderpals/ui/screens/suggestion/SuggestionDetail.kt

```diff
@@ -359,3 +309,35 @@ private fun onDone(
     }
   }
 }
+
+/**
+ * DetailRow composable function to display the details of a suggestion.
+ *
+ * @param icon The icon to display.
+ * @param contentDescription The content description of the icon.
+ * @param text The text to display.
+ * @param color The color of the text.
+ * @param testTag The test tag of the text.
+ */
+@Composable
+fun DetailRow(
```

Comment:
Since it's only used in this file, I'm completely fine leaving it inside of this file. Otherwise, it should have been moved to a separete one